redis-namespace.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.files            += Dir.glob("test/**/*")
   s.files            += Dir.glob("spec/**/*")
 
-  s.add_dependency    "redis", "~> 3.0", ">= 3.0.4"
+  s.add_dependency    "redis", ">= 3.0.4"
```

Comment:
Might be better to specify a range of compatibility:
`s.add_dependency    "redis", ">= 3.0.4", "< 5.0"`